sources/packages/web/src/views/institution/locations/program-info-request/LocationEditProgramInfoRequest.vue

```diff
@@ -145,6 +145,10 @@ export default defineComponent({
         programInfoRequestStatus: mapProgramInfoChipStatus(
           programRequestData.value.pirStatus,
         ),
+        selectedProgram:
+          !isReadOnly.value || !programRequestData.value.isActiveProgram
```

Comment:
It must be &&

`!isReadOnly.value || !programRequestData.value.isActiveProgram`